addons/medical_engine/XEH_postInit.sqf

```diff
@@ -94,3 +94,8 @@
         };
     };
 }] call CBA_fnc_addEventHandler;
+
+["CAManBase", "Killed", {
+    params ["_unit"];
+    [_unit] call FUNC(unlockUnconsciousSeat);
```

Comment:
This kinda defeats the point of #7959 IMO. Better to get the unloading working at 100%.